docs/data-collection.md

```diff
@@ -162,6 +162,10 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cam_app_workload_migrations tracks number of app workload migrations
   // by current state. Tracked migration states are idle, running, completed, and failed.
   '{__name__="cam_app_workload_migrations"}',
+  // apiserver_current_inflight_requests refers to maximal number of currently used
+  // inflight request limit of an apiserver per request kind in last second.
```

Comment:
In last second? What do you mean here?

Reply:
updated the comment, let me know if this is clear.